jenkins/bootstrap.py

```diff
@@ -758,12 +760,24 @@ def setup_credentials(call, robot, upload):
             'Create service account and then create key at '
             'https://console.developers.google.com/iam-admin/serviceaccounts/project',  # pylint: disable=line-too-long
         )
-    call([
-        'gcloud',
-        'auth',
-        'activate-service-account',
-        '--key-file=%s' % os.environ[SERVICE_ACCOUNT_ENV],
-    ])
+    # this sometimes fails spuriously due to DNS flakiness, so we retry it
```

Comment:
hmm it seems my editor also auto pep8 formatted the file.  this block here is the actual change...